verl/utils/reward_score/prime_code/__init__.py

```diff
@@ -50,7 +50,7 @@ def compute_score(completion, test_cases, continuous=False):
             metadata_list = []
             res_list = []
             for test_case_id, test_case in enumerate(test_cases_list):
-                res, metadata = apps_check_correctness(in_outs=test_case, generation=solution, timeout=5, debug=False)
+                res, metadata = apps_check_correctness(in_outs=test_case, generation=solution, timeout=10, debug=False)
```

Comment:
why changing the timeout? What is the unit used here (second, minute?)

Reply:
It is because the local sandbox changed timeout from 10 to 5. Using the same default value to have the same effect.